docs/topics/proc_configuring-jenkins-to-deploy-an-application-with-fmp.adoc

```diff
@@ -0,0 +1,40 @@
+= Configuring Jenkins to deploy your application with the Fabric8 Maven Plugin
+
+Similar to using maven and the Fabric8 Maven Plugin from your local host to deploy an application, you can configure Jenkins to use maven and the Fabric8 Maven Plugin to deploy an application.
+
+[discrete]
+== Prerequisites
+* Access to an OpenShift cluster.
+* Jenkins installed and configured to access the same OpenShift cluster.
+* JDK and Maven installed and configured on your Jenkins server.
+* An application available in GitHub and configured to use maven and the Fabric8 Maven Plugin.
+
+[discrete]
+== Procedure 
+. Create a new OpenShift project for your application.
+
+. Ensure Jenkins has access to that project.
++
+For example, if you configured a service account for Jenkins, ensure that account has `edit` access to the project of your application.
+
+. Create a new link:https://wiki.jenkins.io/display/JENKINS/Building+a+software+project#Buildingasoftwareproject-Settinguptheproject[freestyle Jenkins project] on your Jenkins server.
+.. Under _Source Code Management_, choose _Git_ and add the GitHub url of your application.
+.. Under _Build_, choose _Add build step_ and select `Invoke top-level Maven targets`.
+.. Add the following to _Goals_:
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+clean fabric8:deploy -Popenshift -Dfabric8.namespace=MY_PROJECT
+----
++
+Ensure `MY_PROJECT` is the name of the OpenShift project for your application.
```

Comment:
I find this formulation a bit unwieldy. What about: _Substitute \`MY_PROJECT\` with the name of the OpenShift project for your application._?